lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -35,6 +35,18 @@ public enum S3SseType
         NONE, S3, KMS
     }
 
+    public enum ObjectCannedAcl
+    {
+        NONE,
+        PRIVATE,
+        PUBLIC_READ,
+        PUBLIC_READ_WRITE,
+        AUTHENTICATED_READ,
+        AWS_EXEC_READ,
```

Comment:
We can remove this, as it only it's only useful for EC2 AMIs, not arbitrary data.